runtime/doc/cmdline.txt

```diff
@@ -379,7 +379,7 @@ terminals)
 
 :keepp[atterns] {command}			*:keepp* *:keeppatterns*
 		Execute {command}, without adding anything to the search
-		history
+		history or modifying the last substitute pattern.
```

Comment:
This should be "substitute string" (as mentioned in `:h :&`), not "substitute pattern", but then it's still not accurate as this only keeps the last substitute string for `:&`, not for `~`, so maybe it should be `last substitute string for |:&|`.

Reply:
thanks, I adjusted it